### PR TITLE
MIGRATION

### PR DESCRIPTION
**Pull request by KrashLeviathan**
Added relevant files that weren't in the `endor` sub-folder from the original repo.

This finishes the migration for splitting Endor from the original [Yggdrasil](https://github.com/hammer-io/yggdrasil/) repository.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "extends": "airbnb-base",
+  "plugins": [
+    "import"
+  ],
+  "rules": {
+    "linebreak-style": "off",
+    "comma-dangle": "off",
+    "no-console": "off",
+    "consistent-return": "off",
+    "semi": "off"
```
Comment (nischayv) [inline .eslintrc.js]: Remove the semi: off. We need semicolons on the backend.

Comment (jackcmeyer) [inline .eslintrc.js]: use this for the .eslintrc file

```json
module.exports = {
  "extends": "airbnb-base",
  "plugins": [
    "import"
  ],
  "rules": {
    "linebreak-style": "off",
    "comma-dangle": "off",
    "no-console": "off",
    "consistent-return": "off"
  },
  "env": {
    "node": true
  }
}
```

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,94 @@
+.idea/
```
Comment (nischayv) [inline .gitignore]: Remove the endor filepath from all of these since they won't be in a folder called endor anymore

File: package.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
```
Comment (jackcmeyer) [inline package.json]: update `repository.url`, `bugs.url`, `homepage`

File: .gitignore (hunk 2 above)
Comment (KrashLeviathan) [reply .gitignore]: Done

Comment (nischayv) [reply .gitignore]: 👍

**Pull request merged**